### DISCUSSION TITLE
Can we add 3D model other than MetaHuman ? i.e, from Character Creator, or DAZ ?

### DISCUSSION THREAD

**Discussion by xtremecreative** [endink/Mediapipe4u-plugin#177] [General]
Can we add 3D model other than MetaHuman ? i.e, from Character Creator, or DAZ ?

Or else how to configure new character from scratch ? can you guide me ?

**Comment by endink** [comment]
Of cause!

refer:

https://opensource-labijie-com.translate.goog/Mediapipe4u-plugin/mocap/quick_start/custom_skeleton.html?_x_tr_sl=zh-CN&_x_tr_tl=en&_x_tr_hl=zh-CN&_x_tr_pto=wapp
